src/main/java/wordle/infra/FileReader.java

```diff
@@ -0,0 +1,33 @@
+package wordle.infra;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import wordle.exception.FileReadFailException;
+
+public class FileReader {
+
+    public List<String> readByLine(String filePath) {
+        if (filePath == null || filePath.isBlank()) {
+            throw new FileReadFailException();
+        }
+
+        try {
+            URL systemResource = getUrl(filePath);
+            return Files.lines(Path.of(systemResource.toURI())).toList();
```

Comment:
`Files.lines()`가 Stream을 반환하기 때문에 close해주어야 한다고 하네요!
> This method must be used within a try-with-resources statement or similar control structure to ensure that the stream's open file is closed promptly after the stream's operations have completed.

try-with-resources 구문으로 아래처럼 close해주면 어떨까 싶습니다!

```suggestion
        try (var lines = Files.lines(Path.of(getUrl(filePath).toURI()))) {
            return lines.toList();
```

Reply:
오호, 그렇군요! 세심한 리뷰 감사합니다. 바로 수정하겠습니닷! 🚤

Reply:
[2cb3d49](https://github.com/woowahan-pjs/java-wordle/pull/38/commits/2cb3d4962c76ac5a7bf3f105bcb0c7691e3ac9f9) 작업완료입니닷 👍